tests/test_eval.py

```diff
@@ -15,7 +19,7 @@ def test_evals(capture):
     assert m.test_eval_failure()
 
 
-@pytest.unsupported_on_pypy3
+@pytest.mark.xfail("info.PYPY and not info.PY2", raises=RuntimeError)
```

Comment:
Do these all need to be strings, if `info` is imported anyway?

Reply:
Yes, if it's a string, pytest knows why it failed. Otherwise, a `reason=` is required.

Reply:
Right, OK, that's valuable, indeed; less syntax highlighting but more info...